src/rgw/rgw_rest.cc

```diff
@@ -509,6 +509,17 @@ void dump_owner(req_state *s, const rgw_user& id, const string& name,
   s->formatter->close_section();
 }
 
+extern void dump_objname(req_state *s, bool encode_key, const char* key, const std::string& val) {
```

Comment:
this is fine, but not static;  a static inline version would be in a header

Reply:
and declared with "static inline" specifiers

Reply:
rather than extern

Reply:
repaired